invenio_communities/assets/semantic-ui/js/invenio_communities/community/frontpage.js

```diff
@@ -51,6 +51,9 @@ class CommunityCard extends Component {
     return (
       <Card fluid href={`/communities/${this.props.community.slug}`}>
         <Image
+          wrapped
+          centered
+          ui={false}
```

Comment:
`ui={false}` removes the `.ui` class from the image, to avoid having the extra styling from `.ui.image`. `wrapped` wraps the image in a `<div>`, so that the image dimensions can be maintained within the `<div>`.